pkg/authz/authorizer.go

```diff
@@ -198,9 +200,21 @@ func (m Authorizer) Authenticate(w http.ResponseWriter, r *http.Request, upstrea
 			if m.AuthRedirectStatusCode > 0 {
 				redirOpts["auth_redirect_status_code"] = m.AuthRedirectStatusCode
 			}
-			//redirOpts["logger"] = m.logger
 
-			m.parseLoginHint(r, redirOpts)
+			if m.LoginHintEnabled {
+				queryHasLoginHint := r.URL.Query().Has("login_hint")
+				var loginHint = ""
+
+				if queryHasLoginHint {
+					loginHint = r.URL.Query().Get("login_hint")
+				} else if !queryHasLoginHint && usr != nil {
+					loginHint = usr.Claims.Email
+				}
+
+				if err := m.handleLoginHint(loginHint, redirOpts); err != nil {
```

Comment:
@PedroDBFlores , with the `loginHint` is in `redirOpts`, you only need to pass `redirOpts`. Also, do not throw an error. Rather use `p.logger.Warn()` to indicate that the hint had an issue.